backend/src/services/emailService.ts

```
@@ -20,54 +21,99 @@ export class EmailService {
+    const missing = required.filter((v) => !process.env[v]);
+    if (missing.length) {
+      console.error(
+        `Email configuration missing: ${missing.join(', ')}`,
+      );
+      return false;
+    }
+    return true;
+  }
+
+  private static getTransporter(): nodemailer.Transporter | null {
+    if (this.transporter) return this.transporter;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)